ksql-engine/src/test/java/io/confluent/ksql/KsqlEngineTest.java

```diff
@@ -337,4 +353,44 @@ public void shouldCloseInternallyCreatedTopicClientOnClose() {
     // Then:
     verify(topicClient);
   }
+
+
+  @Test
+  @SuppressWarnings("unchecked")
+  public void shouldParseMultipleStatements() throws IOException {
+    final String statementsString = new String(Files.readAllBytes(
+        Paths.get("src/test/resources/SampleMultilineStatements.sql")), "UTF-8");
+
+    final MetaStore emptyMetaStore = new MetaStoreImpl(new TestFunctionRegistry());
+    final List<PreparedStatement> parsedStatements =
+        ksqlEngine.parseStatements(statementsString, emptyMetaStore, false);
+    assertThat(parsedStatements.size(), equalTo(7));
```

Comment:
assert not needed. The `assertThat(statements, contains(...))` already checks the count.

Reply:
Removed the assert.